examples/ansible.cfg

```diff
@@ -251,11 +251,11 @@
 
 
 # retry files
-# When a playbook fails by default a .retry file will be created in ~/
-# You can disable this feature by setting retry_files_enabled to False
+# When a playbook fails a .retry file can be created that will be placed in ~/
+# You can enable this feature by setting retry_files_enabled to True
 # and you can change the location of the files by setting retry_files_save_path
 
-#retry_files_enabled = False
+retry_files_enabled = False
```

Comment:
You probably want to change the actual value here https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1609 instead.

Reply:
There is a proposal for doing so, but it's not approved yet.

Reply:
Got it.  modified that as well.

Reply:
config file should not have any active values, please revert and add comment back

Reply:
By "add comment back" you are referring to the '#' at the beginning of the line and to
the comments above, correct?  Those were modified as well to account for this change
now defaulting to False.  Would it also make sense to change this line to state

`#retry_files_enabled = True`

to reflect the opposite of the proposed change?